adserver/migrations/0100_add_offer_domain.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 5.0.9 on 2024-12-02 23:42
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('adserver', '0099_link_advertiser_guide'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='click',
+            name='domain',
+            field=models.CharField(blank=True, max_length=10000, null=True, verbose_name='Domain'),
+        ),
+        migrations.AddField(
```

Comment:
We might need to migrate this manually because of the issue with `db_table`.

Reply:
We might. However, the migration file is correct. Worst case we can use `sqlmigrate` and alter it.

```
./manage.py sqlmigrate adserver 0100
```